src/clincoded/static/components/create_gene_disease.js

```diff
@@ -324,6 +328,9 @@ var CreateGeneDisease = createReactClass({
         const moiKeys = Object.keys(modesOfInheritance);
         let gdm = this.state.gdm;
         const submitBusy = this.state.submitBusy;
+        const allowToCreateGDM = isUserAllowedToCreateGdm(this.props.session, this.props.affiliation);
+        const submitClass = 'btn-default pull-right ' + (allowToCreateGDM ? '' : ' disabled');
+        const submitErrClass = 'submit-err pull-right ' + (allowToCreateGDM ? ' hidden' : '');
```

Comment:
I think the error message would look better if it was right-aligned with the submit button. If you add a `margin-right: -15px` it will align, since that is what the `row` class does for the other elements in the form.